### ISSUE TITLE
Instance.watchdog_enabled is missing

### ISSUE THREAD

**Issue by enbeec** [linode/linode_api4-python#275]
### Package

5.3.0

### Expected Behavior

From the docs it looked like I'd be able to read (and hopefully write) the `watchdog_enabled` setting for an instance.

### Actual Behavior

It does not appear that the attribute for `watchdog_enabled` exists at all.

Ideally I'd be able to read and write this setting -- in my use case I currently have a one-off `subprocess.run` invocation of `linode linodes update --watchdog_enabled` as a workaround but that's obviously not ideal.

### Steps to Reproduce

```python
from linode_api4 import LinodeClient
from os import getenv
c = LinodeClient(getenv("LINODE_PAT"))
myInstance = c.linode.instances()[0]
print(myInstance.watchdog_enabled)
```

### Error Output

```python
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
>   File "/home/val/linode-scripts/venv/lib/python3.10/site-packages/linode_api4/objects/base.py", line 135, in __getattribute__
>   return object.__getattribute__(self, name)
> AttributeError: 'Instance' object has no attribute 'watchdog_enabled'
```

**Comment by lgarber-akamai** [comment]
Thanks for the report @enbeec!

The `watchdog_enabled` instance property has been added in [this PR](https://github.com/linode/linode_api4-python/pull/233) and will be made available in our next release (May 1st).

Let us know if you run into any other issues 🙂

**Comment by enbeec** [comment]
Addressed in 5.4.0 (#278)
